README.md

```diff
@@ -23,6 +23,26 @@ zongji.start({
 });
 ```
 
+## Subscribe Binary Log Position
```

Comment:
Instead of a section here, create a section just above `**Options Available:**` for `**Events Emitted:**` and take out the current text for the `on` method to bind handlers:

> Add a listener to the `binlog` or `error` event. Each handler function accepts one argument.